mev-relay-rs/src/relay.rs

```diff
@@ -203,6 +203,7 @@ pub struct Inner {
     beacon_node: ApiClient,
     context: Context,
     state: Mutex<State>,
+    pub genesis_validators_root: Root,
```

Comment:
why is this `pub`?

Reply:
i mistakenly assumed it needed to be pub for the standalone function to access it.